src/main/java/edu/kis/powp/jobs2d/command/gui/CommandHistoryWindow.java

```diff
@@ -0,0 +1,147 @@
+package edu.kis.powp.jobs2d.command.gui;
+
+import edu.kis.powp.appbase.gui.WindowComponent;
+import edu.kis.powp.jobs2d.command.CommandReader;
```

Comment:
Is this import used? Why is it here?

Reply:
Yes, it's used. It's implemented WindowComponent. Realated CommandHistoryFeature.java

Reply:
So is it used or not? I see that it is removed now... ?